suites/portal/GWASUITest.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable codeceptjs/no-skipped-tests */
+Feature('GWAS UI @requires-portal @requires-argo-wrapper @requires-cohort-middleware');
+
+const { Bash } = require('../../utils/bash.js');
+
+const bash = new Bash();
+// const { expect } = require('chai');
+const GWASTasks = require('../../services/portal/GWAS/GWASTasks.js');
+const GWASProps = require('../../services/portal/GWAS/GWASProps.js');
+const GWASQuestions = require('../../services/portal/GWAS/GWASQuestions.js');
+
+async function getRunId() {
+  const res = await bash.runCommand('argo -n argo list | sed -n \'2 p\'');
```

Comment:
We can get the job id from the job list. I would like to get it from argo command just in case the new job is not started successfully or it doesn't show in the list.